src2_object_detection/Box bounding_box
stereo_msgs/DisparityImage disparity_image 
sensor_msgs/Image left_image
sensor_msgs/Image rigt_image
---
geometry_msgs/PointStamped object_position
